assets/js/blocks/reviews/frontend-container-block.tsx

```diff
@@ -4,51 +4,71 @@
 import { __, _n, sprintf } from '@wordpress/i18n';
 import { speak } from '@wordpress/a11y';
 import { Component } from '@wordpress/element';
-import PropTypes from 'prop-types';
 
 /**
  * Internal dependencies
  */
 import { getSortArgs } from './utils';
 import FrontendBlock from './frontend-block';
+import { ReviewBlockAttributes } from './attributes';
+import { Review } from '../../base/components/reviews/types';
```

Comment:
NIT: We can update this import in this way:
```suggestion
import { Review } from '@woocommerce/base-components/reviews/types';
```